roles/provisioner/virsh/tasks/create_vms.yml

```diff
@@ -0,0 +1,78 @@
+---
+- name: set fact for node
+  set_fact:
+      node_data: "{{ item.value }}"
+      node_name: "{{ item.key }}"
+
+- name: import vm disk image(s) for {{ node_name }}
```

Comment:
`import ... for node: "{{ node_name }}"`

So it's clear we have some auto insert here